src/styles/table.scss

```diff
@@ -137,6 +140,39 @@ $block: #{$fd-namespace}-table;
 
   /****** Elements ******/
 
+  &__empty {
+    &-table-message {
+      padding: 1rem;
+      text-align: center;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      align-content: center;
+      width: 100%;
+    }
+
+    &:not(.fixed-height) .fd-table__row:last-of-type {
```

Comment:
I don't know where the class `.fixed-height` is coming from but it doesn't have the correct naming. It should follow BEM and be prefixed with `fd-`

Reply:
done